### PR TITLE
Mbm 69: illuminant steel ingot crafting recipes

### PR DESCRIPTION
**Pull request by YonatanWN**
Naming conventions okay?

### CODE REVIEW

File: src/main/resources/data/minebound/recipes/illuminant_steel_ingot_from_alloy_furnace.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "minebound:alloy_furnace",
+  "ingredients": [
+    {
+      "item": "minecraft:glowstone_dust"
+    },
+    {
+      "item": "minebound:energized_iron_ingot"
+    },
+    {
+      "item": "minecraft:coal"
```
Comment (insertStringHere) [inline src/main/resources/data/minebound/recipes/illuminant_steel_ingot_from_alloy_furnace.json:11]: use the coals tag:
`"tag":"#minecraft:coals"`

**Review by insertStringHere — CHANGES_REQUESTED** [review]

File: src/main/resources/data/minebound/recipes/illuminant_steel_ingot_from_alloy_furnace.json (hunk 1 above)
Comment (MattKirchh) [reply src/main/resources/data/minebound/recipes/illuminant_steel_ingot_from_alloy_furnace.json:11]: If I remember correctly, you don't need the `#` for tags.

**Comment by MattKirchh** [comment]
> Naming conventions okay?

I think appending `_ingot` to the crafting table recipe probably makes sense. So we'll eventually have `illuminant_steel_ingot.json` and `illuminant_steel_block.json`.

**Review by MattKirchh — APPROVED** [review]

**Review by insertStringHere — APPROVED** [review]
Tested

**Pull request merged**